xcode/Subconscious/Shared/Components/Detail/UserProfileDetailView.swift

```diff
@@ -261,6 +261,7 @@ struct UserProfileDetailModel: ModelProtocol {
     var isUnfollowConfirmationPresented = false
     var isEditProfileSheetPresented = false
     
+    var address: Slashlink? = nil
```

Comment:
Address is already part of `UserProfile`. Should we look for it there instead?

Keeping properties in sync with update pattern is no problem, but I try to avoid it unless needed.

Reply:
This is to deal with the fact that (in a "Not Found" state) we do not have access to any user's profile so we need to track the address we attempted to find separately to let us render it.

I am thinking of this in terms of: `address` is where you are trying to navigate, `user` is only present if `address` resolves. Further, the `address` could be `@alice.bob.charlie` _but_ when resolving we might realise we know `@alice` already, so the logical value for `user.address` is `@alice` _not_ `@alice.bob.charlie`.

Reply:
That totally makes sense, and the current solution WFM.